src/sentry/static/sentry/app/views/settings/incidentRules/ruleConditionsFormWithGuiFilters.tsx

```diff
@@ -260,4 +320,36 @@ const SearchEventTypeNote = styled('div')`
   user-select: none;
 `;
 
+const SectionTitle = styled('div')`
+  font-size: ${p => p.theme.fontSizeExtraLarge};
+  margin: ${space(2)} ${space(2)} 0 ${space(2)};
+  display: flex;
+  align-items: center;
+`;
+
+const SectionNumber = styled('div')`
```

Comment:
Can we re-use anything in https://storybook.sentry.dev/?path=/story/core-list--default

Reply:
haha didn't even know this type of list existed
probably should do some more exploring around storybook 😅